### PR TITLE
PXB-3237 Update xbcrypt options (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch pxb-3237-8.0
	modified:   docs/xbcrypt-options.md

### CODE REVIEW

File: docs/xbcrypt-options.md [hunk 1]
```diff
@@ -1,56 +1,70 @@
 # The xbcrypt command-line options
 
-The `xbcrypt` binary has the following command line options:
-
-### encrypt-algo
+Usage: 
+ 
+ ```{.bash data-prompt=text}
```
Comment (alina-derkach-oaza) [inline docs/xbcrypt-options.md]:
```suggestion
```{.bash data-prompt="$"}
```

Comment (alina-derkach-oaza) [reply docs/xbcrypt-options.md]: This code block is broken in the preview.

Comment (patrickbirch) [reply docs/xbcrypt-options.md]: changed to ```{.text .no-copy}
The code is pseudo-code and should not work.

File: docs/xbcrypt-options.md [hunk 2]
```diff
@@ -1,56 +1,70 @@
 # The xbcrypt command-line options
 
-The `xbcrypt` binary has the following command line options:
-
-### encrypt-algo
+Usage: 
+ 
+ ``````{.text .no-copy}
```
Comment (alina-derkach-oaza) [inline docs/xbcrypt-options.md]:
```suggestion
 ```{.text .no-copy}
```

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**